docs/RELEASE_GUIDELINES.md

```diff
@@ -0,0 +1,83 @@
+
+## Security
+
+Due to the restrictions on this repository, and the inability to enable Actions to create pull requests, this workflow uses a personal access token.
+
+You can generate a personal access token by going to your [profile](https://github.com/settings/profile) > Developer settings > Personal access tokens.
+
+- Token name: `RELEASE_PLEASE_TOKEN`
+- Expiration: `90 days`
+- Resource Owner: `Azure-Samples`
+- Repository access: `Only select repositories` > `chat-with-your-data-solution-accelerator`
+
+The personal access token must be regenerated every 90 days. It must have the following permissions in order for the GitHub Action to be able to create a release pull request:
+- Actions: `Read and write`
+- Contents: `Read and write`
+- Merge queues: `Read and write`
+- Metadata: `Read-only`
+- Pull requests: `Read and write`
+- Workflows: `Read and write`
+
+Once the personal access token has been generated, it should be stored in the repository Settings, under Security > Secrets and Variables > Actions > RELEASE_PLEASE_TOKEN.
+
+# Conventional Commits
+
+This is a [specification](https://www.conventionalcommits.org/en/v1.0.0/) which is commonly used to help communicate the nature of the changes in a pull request or commit. As in our case, it makes it easier to use automated tooling based on those namings. For this repository, we ask the PRs should be prefixed with one of the following `types`:
+  * feat: A new feature
+  * fix: A bug fix
+  * docs: Documentation only changes
+  * style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+  * refactor: A code change that neither fixes a bug nor adds a feature
+  * perf: A code change that improves performance
+  * test: Adding missing tests or correcting existing tests
+  * build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+  * ci: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
+  * chore: Other changes that don't modify src or test files
+  * revert: Reverts a previous commit
+
+
+# Semantic Versioning
+
+[Semantic Versioning (SemVer)](https://semver.org/) provides a set of rules to determine what version number a piece of software should have.
+
+The guidance states:
+
+> Given a version number MAJOR.MINOR.PATCH, increment the:
+>
+> MAJOR version when you make incompatible API changes
+> MINOR version when you add functionality in a backward compatible manner
+> PATCH version when you make backward compatible bug fixes
+> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+
+## What should I do if I introduce a breaking change?
+
+If you must introduce a breaking change, then the Pull Request should clearly indicate this using the `BREAKING CHANGE` prefix. This will correlate to a `major` semantic version increase.
```

Comment:
Where is `BREAKING CHANGE` documented? I can only see `!` for requiring breaking changes

Reply:
You are 100% correct, thanks for catching this. Will update the documentation, I've just tested out using `feat!` on my test repo and it picked it up as a breaking change. Cheers!